src/ethereum_test/types.py

```diff
@@ -13,7 +13,33 @@
 from .common import AddrAA, TestPrivateKey
 
 
-def hex_or_none(input: Union[int, None], default=None) -> Union[str, None]:
+class UNSPECIFIED(object):
```

Comment:
Could you expand on why this class is necessary? I _think_ we should be able to generally work around this by having different methods for creating objects that set the defaults to non-`None` values and if a test writer desires `None` actually being specified they can use a more general initializer that accepts `None`?

Reply:
Also, this may just be due to my ignorance, but I would be curious to know if this a pattern you've seen before. I'm not as familiar with larger-scale python projects, maybe this is common.

Reply:
I actually copied the idea from the `dataclasses` library `MISSING` sentinel object over here: https://docs.python.org/3/library/dataclasses.html#dataclasses.field.

I think it's definitely possible to skip this pattern completely, so I think it's just a matter of deciding how we want the test definitions to look like, and here are some examples of how a tester could specify an Account's code in post and specifically define if they want it verified or not:

Without `UNSPECIFIED`:
```
Account()
- code is not checked

Account(code=None)
- code is not checked

Account(code="0x")
- code is expected to be empty
```
With `UNSPECIFIED`:
```
Account()
- code is not checked

Account(code=None)
- code is expected to be empty

Account(code="0x")
- code is expected to be empty
```
 Where `code is not checked` means that the `tf` tool will not check what was received in the transition tool and simply pass it along onto the fixture.

Reply:
Thanks for all this extra context!! I do like the new `NONEXISTENT` approach.